internal/elasticsearch/index/index.go

```diff
@@ -519,6 +520,32 @@ If specified, this mapping can include: field names, [field data types](https://
 			Type:        schema.TypeString,
 			Computed:    true,
 		},
+		"include_type_name": {
+			Type:        schema.TypeBool,
+			Description: "If true, a mapping type is expected in the body of mappings. Defaults to false.",
```

Comment:
We should detail the version constraint here
```suggestion
			Description: "If true, a mapping type is expected in the body of mappings. Defaults to false. Supported for Elasticsearch 7.x",
```